src/test/java/com/redhat/devtools/lsp4ij/LSPIJUtils_toOffsetTest.java

```diff
@@ -74,6 +75,20 @@ public void testInvalidPositionWithTwoLine() {
         assertOffset("foo\nbar", 999999, 999999, 7, null);
     }
 
+    public void testPureOffsetPositions() {
+        String content = "foo\nbar\nbaz";
+        for (int character = 0; character < content.length(); character++) {
+            assertOffset(content, 0, character, character, content.charAt(character));
+        }
+    }
+
+    public void testInvalidPureOffsetPositions() {

```

Comment:
Negative test that confirms that `line=0, column=<invalid>` results in the respective bounds of the content.